app/controllers/items_controller.rb

```diff
@@ -0,0 +1,54 @@
+class ItemsController < ApplicationController
+
+	before_action :set_todo
+	before_action :set_todo_item, only: [:show, :update, :destroy]
+
+	#GET /todos/:todo_id/items
+	def index
+		json_response(@todo.items)
+	end
+
+	def create
+		@item = @todo.items.new(item_params)
+		if @item.save!
+			json_response(@item, :created)
+		else
+			render json: { status: :unprocessable_entity, message: @item.errors.full_messages.first} 
```

Comment:
This is not good approach. We should use
`json_response({ message: @todo.errors.full_messages }, :unprocessable_entity)`